app/src/test/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupAutoFillViewModelTest.kt

```diff
@@ -0,0 +1,114 @@
+package com.x8bit.bitwarden.ui.auth.feature.accountsetup
+
+import app.cash.turbine.test
+import com.google.common.base.Verify.verify
+import com.x8bit.bitwarden.data.platform.repository.SettingsRepository
+import com.x8bit.bitwarden.ui.platform.base.BaseViewModelTest
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class SetupAutoFillViewModelTest : BaseViewModelTest() {
+
+    private val mutableAutoFillEnabledStateFlow = MutableStateFlow(false)
+    private val settingsRepository = mockk<SettingsRepository>(relaxed = true) {
+        every { isAutofillEnabledStateFlow } returns mutableAutoFillEnabledStateFlow
+        every { disableAutofill() } answers {
+            mutableAutoFillEnabledStateFlow.value = false
```

Comment:
Doesn't this overcomplicate the test. Could this just be `every { disableAutofill() } just runs`

Reply:
I thought this was pretty clever and allowed for a little more full circle test, but I can nuke it so the test is just the unit of "is the mock called"